#CREATE request parameters
int8 mode
string[] names
float32[] angles
float32 max_speed
float32[] time_second

--------------------------------
#CREATE response parameters
bool reply
